### ISSUE TITLE
Screen reader does not announce the current value of status bar items when the context menu for the status bar is opened in word.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#18457]
### Steps to reproduce:

1. Open Microsoft Word.
2. Press F6 to move focus to the status bar.
3. Press the Application key (or Shift + F10) to open the context menu for status bar items.
4. Navigate through the list of status bar items (such as "Page number," "Word Count," "Section," etc.).
5. Observe that screen reader announces the current value of status bar items when the context menu for the status bar is opened in word or not

### Actual behavior:
The context menu shows the status bar items as checked or unchecked but does not announce the current values of these items (e.g., "Page 2 of 10" for Page number, "150 words" for Word count).

### Expected behavior:
The context menu should announce both the checked/unchecked state of the status bar items and their current values (e.g., "Page number, checked, Page 2 of 10," "Word count, checked, 150 words").

### System configuration
OS: Windows 11 Version 24H2 (OS Build 26100.4484)
App: Microsoft Word (2507 Build 16.0.19019.20000)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2025.1.2

#### Windows version:
Win 11

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

### Other questions
N/A
#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Refer Attachment:**
1.Screen reader does not announce the current value of status bar items when the context menu for the status bar is opened in word.png
2.Screen reader does not announce the current value of status bar items when the context menu for the status bar is opened in word.mp4

https://github.com/user-attachments/assets/5ce5f9d3-99a0-4088-a6e6-dc4899fac167
<img width="966" height="594" alt="Image" src="https://github.com/user-attachments/assets/b70fc95c-1020-414f-987b-8547ec893658" />

**Comment by Qchristensen** [comment]
This is a duplicate of #5996.  At the time it was reported (2016) I opened an issue for Microsoft on their UserVoice platform, but that has since been deleted

**Comment by SaschaCowley** [comment]
The current value is read when mousing over the menu items if mouse tracking is enabled in NVDA.
Narrator does not report the current value.

**Comment by msftedad** [comment]
hI @SaschaCowley, Issue is fixed now.

**Comment by seanbudd** [comment]
Thanks @msftedad
